lib/model.d.ts

```diff
@@ -325,7 +325,7 @@ export interface IncludeOptions {
   /**
    * Load further nested related models
    */
-  include?: Includeable[];
+  include?: Includeable[] | IncludeAll[];
```

Comment:
Actually, `IncludeOptions` is already used in `Includeable` for complex include options. Renaming the existing typing would break backwards compatibility and could potentially confuse people.

How about `IncludeableOptions` or something similar?